en-US/E.xml

```diff
@@ -104,6 +104,22 @@
 
 		 </listitem>
 
+		 </varlistentry>
+		<varlistentry id="epub">
+		 <term>EPUB</term>
+			<listitem>
+			 <para>
+				Write the term unhyphenated, and without any variations in styling or capitalization, to refer to this publishing format.
+				 Do not use EPUB as a noun to refer to an EPUB file. 
+				 For example, instead of writing "in an EPUB", write "in an EPUB file". 
+			 </para>
+			 <para>
+			 EPUB is part of the w3.org publishing standards: 
+			 <ulink url="https://www.w3.org/publishing/epub3/">https://www.w3.org/publishing/epub3/</ulink>
+			 </para>
```

Comment:
Yes, the link will eventually need updating but it points to a major version, and based on history it'll be a long time before we see v.4. I'm ok with it.